l10n_lu_fin_rep_details/account_financial_report.xml

```diff
@@ -410,5 +410,21 @@
             <field name="display_detail">detail_flat</field>
         </record>
 
+        <record model="account.financial.report" id="l10n_lu.account_financial_report_162">
+            <field name="display_detail">detail_flat</field>
+        </record>
+
+        <record model="account.financial.report" id="l10n_lu.account_financial_report_163">
+            <field name="display_detail">detail_flat</field>
+        </record>
+
+        <record model="account.financial.report" id="l10n_lu.account_financial_report_164">
+            <field name="display_detail">detail_flat</field>
+        </record>
+
+        <record model="account.financial.report" id="l10n_lu.account_financial_report_165">
+            <field name="display_detail">detail_flat</field>
+        </record>
```

Comment:
@laetitia-gangloff I'm not sure we want the details for these, since the details should already appear in other sections. Let's check on a concrete example.